WebContent/WEB-INF/jsp/viewEdit.jsp

```diff
@@ -385,7 +385,20 @@
                   <td class="formError">${status.errorMessage}</td>
                 </tr>
               </spring:bind>
+              <spring:bind path="form.view.id">
+                <tr>
+                  <td class="formField" width="250">
+                    <input type="hidden" id="id" name="view.id" value="${status.value}"/>
+                  </td>
+                </tr>
+              </spring:bind>
 
+                <tr>
+                  <td class="formField" width="250">
+                    <input type="hidden" id="dwrScriptSessionid" name="dwrScriptSessionid" value="">
+                  </td>
+                  <td class="formError">${status.errorMessage}</td>
```

Comment:
probably it is not needed "<td class="formError">${status.errorMessage}</td>"

Reply:
ok, thank you